Parsers/ASim WindowsEvent/ARM/MicrosoftWindowsEventFullDeployment.json

```diff
@@ -82,10 +82,10 @@
                     "contentVersion": "1.0.0.0"
```

Comment:
Might be a review issue, but I think that vimAuthentication is not included.

Reply:
not a review issue. you are right. The existing ASim Resource name is  VIM. to avoid generating conflicts, duplication on early adopters env. I will not fix this